vite.config.js

```diff
@@ -1,7 +1,17 @@
 import { defineConfig } from 'vite'
+import dotenv from 'dotenv';
 import react from '@vitejs/plugin-react'
 
+dotenv.config(); // load env vars from .env
+
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
+  define: {
+    API_BASE_URL: `"${process.env.API_BASE_URL}"`,
```

Comment:
this is to set the API_BASE_URL used in src/api

no need to use the import.meta anymore

Reply:
@muhammadiqbalid83